src/components/Reusables/ConfirmationPage/ConfirmationPage.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  Page,
+  PageContent,
+  PageHeader,
+  List,
+  BorderStyle,
+  PageColor,
+} from '@altinn/altinn-design-system';
+import type { Key } from 'react';
+
+import type { DelegableApi } from '@/rtk/features/delegableApi/delegableApiSlice';
+import { softRemove as softRemoveApi } from '@/rtk/features/delegableApi/delegableApiSlice';
+import { softRemove as softRemoveOrg } from '@/rtk/features/delegableOrg/delegableOrgSlice';
+import { useAppDispatch } from '@/rtk/app/hooks';
+import type { DelegableOrg } from '@/rtk/features/delegableOrg/delegableOrgSlice';
+import { ReactComponent as OfficeIcon } from '@/assets/Office1.svg';
+import { ReactComponent as SettingsIcon } from '@/assets/Settings.svg';
+import { CompactDeletableListItem } from '@/components/Reusables/CompactDeletableListItem';
+
+import classes from './ConfirmationPage.module.css';
+
+export interface ConfirmationPageProps {
+  firstListItems: DelegableApi[];
+  secondListItems: DelegableOrg[];
+  headerText: string;
+  firstContentText?: string;
+  secondContentText?: string;
+  bottomText?: string;
+  mainButton?: React.ReactNode;
+  complementaryButton?: React.ReactNode;
```

Comment:
Why is this called complimentaryButton when it is of class previousButton?

Can't it just be a backButton or previousButton?

Reply:
Thought we could generalize it if we want it to be something else than backButton in the future

Reply:
But in that case, the code should probably be generalized too.
Right now it has a classname of previousButton and it lies in a dev called navButtonSection.